### DISCUSSION TITLE
command mode autocomplete visual text leak issue

### DISCUSSION THREAD

**Discussion by kapral18** [LazyVim/LazyVim#1604] [Q&A] [answered]
Whenever I invoke autocomplete in command mode, I get this weird underlying letters
visually leaking into the autocomplete field making it hard to read the completion box

<img width="522" alt="CleanShot 2023-10-06 at 20 52 45@2x" src="https://github.com/LazyVim/LazyVim/assets/1625373/d3ac212e-ff8f-4b9d-9d3f-25d28be57fbb">

My current config: https://github.com/kapral18/config.nvim/blob/7327cd3005c3572748bd3153001a09b02e696f03/lua/config/lazy.lua

Any ideas?

**Comment by folke** [comment, marked as answer]
set `vim.opt.pumblend = 0` in your `lua/config/options.lua`

**Reply by kapral18** [reply]
thank you!
